app/page.tsx

```diff
@@ -8,8 +8,8 @@ const HomePage = () => {
   const router = useRouter();
 
   useEffect(() => {
-    router.push(PAGE_ROUTES.Dashboard);
-  });
+    router.push(PAGE_ROUTES.Signin);
+  }, []);
```

Comment:
Remove the dependency array because it is empty and it will produce a warning while building.